core/core-funcs.el

```diff
@@ -119,6 +119,14 @@ Supported properties:
   (re-search-forward anchor-text)
   (beginning-of-line)
   (show-subtree)
+  (setq-local org-emphasis-alist '(("*" bold)
+                                   ("/" italic)
+                                   ("_" underline)
+                                   ("=" org-verbatim verbatim)
+                                   ("~" org-kbd)
+                                   ("+"
+                                    (:strike-through t))))
+
```

Comment:
`SPC f e h` still loads the rendered markdown though

On Tue, Jun 9, 2015 at 5:33 AM, Tu Do <notifications@github.com> wrote:

> In core/core-funcs.el
> <https://github.com/syl20bnr/spacemacs/pull/1906#discussion_r32000263>:
>
> > @@ -119,6 +119,14 @@ Supported properties:
> >    (re-search-forward anchor-text)
> >    (beginning-of-line)
> >    (show-subtree)
> > +  (setq-local org-emphasis-alist '(("*" bold)
> > +                                   ("/" italic)
> > +                                   ("_" underline)
> > +                                   ("=" org-verbatim verbatim)
> > +                                   ("~" org-kbd)
> > +                                   ("+"
> > +                                    (:strike-through t))))
> > +
>
> Yes, it can be used since I updated the opening function for SPC f e h
> and it was merged.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/syl20bnr/spacemacs/pull/1906/files#r32000263>.
>